src/reference/changes.rst

```diff
@@ -132,6 +132,41 @@ The graph view now has an option to group tasks by cycle point.
 .. image:: changes/cylc-graph-group-by-cycle-point.png
    :width: 100%
 
+
+Completion Expressions
+^^^^^^^^^^^^^^^^^^^^^^
+
+When a task finishes, its outputs are validated against a "completion
+expression" to ensure that it has produced all of its
+:term:`required outputs <required output>`.
+If a task fails this validation check it is said to have "incomplete outputs"
+and will be retained in the :term:`active window` pending user intervention.
+
+This completion expression is generated automatically from the graph.
+By default, tasks are expected to succeed, if you register any additional
+required output in the graph, then these must also
+be produced.
+
+At Cylc 8.3.0 it is now possible to manually configure this completion
```

Comment:
No biggie, but we'd normally not say this here (put an "added in version ..." note instead?)

Reply:
```suggestion
.. versionadded:: 8.3.0

   It is now possible to manually configure this completion
```

Reply:
I'm not sure we'd do that in `changes.rst` - or the whole document would be a storm of `.. versionadded::` directives.